spec/models/metric/capture_spec.rb

```diff
@@ -10,46 +10,40 @@
                      :capture_threshold_with_alerts => {:vm => capture_rt, :host => capture_rt}
                     }
                   }
-      stub_server_configuration(settings)
+      stub_settings(settings)
     end
 
-    it "realtime vm uses capture_threshold_with_alerts minutes ago" do
+    it "knows realtime vm capture value" do
```

Comment:
Yeah, the names are terrible here too (and I see where the terms "realtime" and "non-realtime" come from previously, but they are bad).  I think this should say "with a vm with alerts", or perhaps do the standard

```ruby
describe ".alert_capture_threshold" do
  it "with a vm with alerts" {};
  it "with a vm without alerts" {};
end
```